graph/db/sql_store.go

```diff
@@ -3092,7 +3094,7 @@ func (s *sqlNodeTraverser) FetchNodeFeatures(nodePub route.Vertex) (
 
 	ctx := context.TODO()
 
-	return fetchNodeFeatures(ctx, s.db, nodePub)
+	return fetchNodeFeatures(ctx, s.db, lnwire.GossipVersion1, nodePub)
```

Comment:
why don't we add a version in the parameters for `FetchNodeFeatures` and `ForEachSourceNodeChannel`?